src/main/java/org/prgrms/cream/domain/product/service/ProductService.java

```diff
@@ -60,9 +75,27 @@ private void modifyOption(Product product, String size) {
 		}
 	}
 
-	private Product findActiveProduct(Long id) {
+	public Product findActiveProduct(Long id) {
```

Comment:
```suggestion
        @Transactional(readOnly = true)
	public Product findActiveProduct(Long id) {
```

@Transactional 어노테이션을 붙여주면 좋을 것 같습니다!

Reply:
아 넵! 상세 조회 기능에서 수정하도록 하겠습니당😆